src/plugins/discover/public/__mocks__/services.ts

```diff
@@ -163,6 +163,7 @@ export function createDiscoverServicesMock(): DiscoverServices {
     data: dataPlugin,
     dataVisualizer: {
       FieldStatisticsTable: jest.fn(() => createElement('div')),
+      getReasonIfFieldStatsUnavailableForQuery: jest.fn().mockReturnValue(undefined),
```

Comment:
```suggestion
```

Can be reverted now, right?

Reply:
Reverted here [`eabc053` (#197538)](https://github.com/elastic/kibana/pull/197538/commits/eabc0531fe65d8f0c85807b82e0d7e182ecb9811)